packages/expo/src/Linking/Linking.ts

```diff
@@ -189,4 +189,9 @@ newLinking.makeUrl = makeUrl;
 newLinking.parse = parse;
 newLinking.parseInitialURLAsync = parseInitialURLAsync;
 
+// TODO: remove this on SDK 38 after adding to EXLinking
+if (Platform.OS === 'ios') {
+  newLinking.openSettings = () => newLinking.openURL('app-settings:');
```

Comment:
[open settings on android](https://github.com/expo/expo/blob/b007650c8df29251cd6760d4d61949d50a69a3fc/home/utils/PermissionUtils.js#L79-L91) is kinda complex

Reply:
yeah this is already implemented and works in expo client, we just didn't fix it on the ios side